modules/ocf_desktop/files/xsession/staff-logout

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+import datetime
+from subprocess import call
+from subprocess import PIPE
+from subprocess import Popen
+from time import sleep
+
+from ocflib.lab.hours import Day
+
+# delay before killing session in minutes
+delay = 45
+# extra delay if we will be closed delay minutes from now
+bonus = 30
+now = datetime.datetime.now()
+if not Day.from_date(now).is_open(now + datetime.timedelta(minutes=delay)):
```

Comment:
This won't work if `now + datetime.timedelta(minutes=delay)` happens to be on the next day. I think you meant `Day.from_date(now + datetime.timedelta(minutes=delay)).is_open(now + datetime.timedelta(minutes=delay))` (of course you can simplify that expression)